tableauserverclient/server/endpoint/sites_endpoint.py

```diff
@@ -59,7 +59,12 @@ def delete(self, site_id):
             raise ValueError(error)
         url = "{0}/{1}".format(self.baseurl, site_id)
         self.delete_request(url)
-        logger.info('Deleted single site (ID: {0})'.format(site_id))
+        # Deleting the site also signs you out on the Server.
+        # Here we check that you're deleting the site you are signed in to
+        # (Which is true now but may not always be) and then clear auth tokens
```

Comment:
This comment doesn't make sense, "Which is true now". Our code does not enforce this. It might be true that this only works in this case but I think you can leave that out. Simply put if you delete the site you are signed into, you are automatically signed out.